yt_dlp/postprocessor/ffmpeg.py

```diff
@@ -586,6 +587,7 @@ def _options(target_ext):
 
 
 class FFmpegEmbedSubtitlePP(FFmpegPostProcessor):
+    AUDIO_EXTS = ('mp3', 'm4a', 'flac', 'opus')
     SUPPORTED_EXTS = ('mp4', 'mov', 'm4a', 'webm', 'mkv', 'mka')
```

Comment:
```suggestion
    SUPPORTS_LYRICS = ('mp3', 'm4a', 'flac', 'opus')
    SUPPORTED_EXTS = ('mp4', 'mov', 'webm', 'mkv', 'mka', *SUPPORTS_LYRICS)
```

`mka` is also audio. So the name AUDIO_EXTS doesn't make sense